h2/src/test/org/h2/samples/CatalogTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2004-2019 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (https://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.samples;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import org.h2.tools.DeleteDbFiles;
+
+/**
+ * A very simple class that shows how to load the driver, create a database,
+ * create a table, and insert some data.
+ */
+public class CatalogTest {
```

Comment:
The test case must be located in `h2/src/test` directory and be invoked from the `TestAll`.

Reply:
Ok, thanks

Reply:
You need to extend and implement the `TestDb`. Take a look on existing test cases.

Reply:
created TestIgnoreCatalogs

Reply:
Please, remove the `samples/CatalogTest.java`. It isn't needed.

Reply:
done